### PR TITLE
CN0409 update

### PR DESCRIPTION
**Pull request by adrimbarean**
Add option to run the program without calibration by inputting calibration values manually.
Make minor changes to improve consistency in the code and between the code and wiki.

### CODE REVIEW

File: projects/ADuCM360_demo_cn0409/src/CN0409.c [hunk 1]
```diff
@@ -371,10 +371,45 @@ void CN0409_TurbidityCalculation(void)
...
 		if (ISORatioCoeff.status != UPDATED ||
-		    ISONonRatioCoeff.status != UPDATED)
+		    ISONonRatioCoeff.status != UPDATED) {
 			/* Calibration is required */
 			AppPrintf("Calibration is required because of "
-				  "invalid coefficients! \n\r");
+				  "invalid coefficients! Please provide calibration "
+				  "coefficients for both ratio and non-ratio measurement.\n\r"
+				  "\tRatio slope: ");
+			while(!uart_enter);
```
Comment (mirceacaprioru) [inline projects/ADuCM360_demo_cn0409/src/CN0409.c]: I think this could be included inside a function.

Comment (mirceacaprioru) [reply projects/ADuCM360_demo_cn0409/src/CN0409.c]: Also there are repetitive pieces of code.

**Comment by adrimbarean** [comment]
V0 -> V1:
 - include added code in functions to prevent repetition and improve readability

**Comment by mirceacaprioru** [comment]
will merge later today if no objections.

**Pull request merged**